io/net/net_data.py

```diff
@@ -161,10 +169,10 @@ def interface_wait(self, cmd):
         '''
          Waits for the interface to come up
         '''
-        for i in range(0, 600, 5):
+        for _ in range(0, 600, 5):
```

Comment:
@apahim Thanks for this. Will try this and send a PR to enhance the code here and in some other tests as well